crypto/armcap.c

```diff
@@ -68,6 +68,12 @@ void OPENSSL_cpuid_setup(void) __attribute__ ((constructor));
 #   include <sys/auxv.h>
 #   define OSSL_IMPLEMENT_GETAUXVAL
 #  endif
+# elif defined(__ANDROID_API__)
+// see https://developer.android.google.cn/ndk/guides/cpu-features
```

Comment:
Please use C style comments: `/* ... */ `not C++ `// ...`

This applies throughout.

Reply:
got it, i will remove this comment and add it into commit

Reply:
I think these comments were useful just like this instead:
```c
/* see https://developer.android.google.cn/ndk/guides/cpu-features */
```

Reply:
ok, i will add them both place